scripts/csv-to-sqlite.py

```
@@ -0,0 +1,165 @@
+        os.remove(DB_PATH)
+        print(f"Removed existing DB: {DB_PATH}")
+
+    con = sqlite3.connect(DB_PATH)
+    cur = con.cursor()
+    cur.execute(CREATE_TABLE)
+    for idx_sql in CREATE_INDEXES:
+        cur.execute(idx_sql)
+
+    cols_ordered = list(COLUMN_MAP.values())
+    placeholders = ', '.join(['?'] * len(cols_ordered))
+    insert_sql = f"INSERT INTO recipients ({', '.join(cols_ordered)}) VALUES ({placeholders})"
```

Ruff (0.15.1)

[error] Possible SQL injection vector through string-based query construction

(S608)